### ISSUE TITLE
💡 Feature request: Ability to quickly create a record (such as taking down a note)

### ISSUE THREAD

**Issue by marks** [Airtable-Labs/raycast-extension#2]
One potential would be to allow the configuration of a specific table and set of fields and have a Raycast command which opens up a Raycast form view with those specific fields.

Supporting the long tail of field types may be difficult, so leaving it to single line text, single select, and long text to start would probably be a good start.

**Comment by egordin** [comment]
Yes! This would be awesome! I use a SSB for forms but this would solve that!
